enable-jquery-migrate-helper.php

```diff
@@ -2,152 +2,24 @@
 /*
 Plugin Name: Enable jQuery Migrate Helper
 Plugin URI: https://wordpress.org/plugins/enable-jquery-migrate-helper
-Description: Enable the jQuery Migrate helper feature during a transitional phase
-Version: 0.1.0
-Author: Clorith
-Author URI: https://www.clorith.net
+Description: Enable support for old and outdated plugins and themes during a jQuery update transitional phase.
+Version: 1.0.0
+Requires at least: 5.5
```

Comment:
Would it make sense to set the requires at least to `5.4` instead, that way those who are vary of updating, or are just pre-emptively adding it as they know there will be a conflict won't be locked out?

Reply:
Yes, makes sense. Then the plugin will replace the version of jQuery Migrate and show the warnings in wp-admin. Can probably disable `dashboard_notice()` in that case.

Reply:
I was thinking leave everything the same for both 5.4 and 5.5, be up front about what may be throwing warnings and could cause issues moving forward.

Reply:
Sure, that works too :)